inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/interpolate.cpp

```diff
@@ -68,7 +67,6 @@ const std::vector<double> cubeCoefs = {
 };
 
 const auto interpolateCases = ::testing::Combine(
-        ::testing::ValuesIn(axes),
         ::testing::ValuesIn(modes),
         ::testing::ValuesIn(coordinateTransformModes),
         ::testing::ValuesIn(nearestModes),
```

Comment:
"nearestModes" is only applicable for "nearest" interpolate mode. Other interpolate mode do not need this parameter. If combine this attribute to other mode, we just increase test time due to run several times for the same test case. Maybe we can have one "INSTANTIATE_TEST_CASE_P()" for nearest mode, and one for other modes. Could you help improve this?

Reply:
Done.